VIT: make so `mock.StateKeyBuilder` implement `fmt.Stringer()` interface

## Steps to reproduce the behavior

bp3: [TestNewRestaurantVatProjector_BasicUsage()](https://github.com/untillpro/airs-bp3/blob/d62dd9668ec185ed968abe1f5df934a98ea6dd4f/packages/air/vat/impl_newrestaurantvatprojector_test.go#L31): 
```
--- FAIL: TestNewRestaurantVatProjector_BasicUsage (0.00s)
panic: interface conversion: *mock.StateKeyBuilder is not istructs.IStateKeyBuilder: missing method String [recovered]
        panic: interface conversion: *mock.StateKeyBuilder is not istructs.IStateKeyBuilder: missing method String

goroutine 7 [running]:
testing.tRunner.func1.2({0x10b6bc0, 0xc00020f950})
        C:/Program Files/Go/src/testing/testing.go:1632 +0x225
testing.tRunner.func1()
        C:/Program Files/Go/src/testing/testing.go:1635 +0x359
panic({0x10b6bc0?, 0xc00020f950?})
        C:/Program Files/Go/src/runtime/panic.go:785 +0x132
github.com/voedger/voedger/pkg/vit/mock.(*State).KeyBuilder(0xc000208a20, {{0x112559f, 0x3}, {0x11260f0, 0x6}}, {{0x0, 0x0}, {0x0, 0x0}})
        C:/workspace/go/gopath/pkg/mod/github.com/voedger/voedger@v0.0.0-20240916130329-a109ed54f540/pkg/vit/mock/impl.go:41 +0x1a9
github.com/untillpro/airs-bp3/packages/air/vat.(*cudRowHandler).findUserProfile(0xc0000000c0, 0x29)
        C:/workspace/go/airs-bp3/packages/air/vat/types.go:68 +0x6b
github.com/untillpro/airs-bp3/packages/air/vat.(*cudRowHandler).handle(0xc0000000c0, {0x11ba308, 0xc000208540})
        C:/workspace/go/airs-bp3/packages/air/vat/types.go:47 +0xdf
github.com/untillpro/airs-bp3/packages/air/vat.TestNewRestaurantVatProjector_BasicUsage.newRestaurantVatProjector.func4.1({0x11ba308?, 0xc000208540?})
        C:/workspace/go/airs-bp3/packages/air/vat/impl_newrestaurantvatprojector.go:40 +0x2a
github.com/voedger/voedger/pkg/goutils/iterate.ForEachError[...].func1()
        C:/workspace/go/gopath/pkg/mod/github.com/voedger/voedger@v0.0.0-20240916130329-a109ed54f540/pkg/goutils/iterate/impl.go:39 +0x30
github.com/untillpro/airs-bp3/packages/air/vat.TestNewRestaurantVatProjector_BasicUsage.func3({0xc0001bd370?, 0x12082b0?, 0x4?})
        C:/workspace/go/airs-bp3/packages/air/vat/impl_newrestaurantvatprojector_test.go:123 +0x3f
github.com/stretchr/testify/mock.(*Mock).MethodCalled(0xc0002085b0, {0x12082b0, 0x4}, {0xc0001bd370, 0x1, 0x1})
        C:/workspace/go/gopath/pkg/mod/github.com/stretchr/testify@v1.9.0/mock/mock.go:554 +0xd33
github.com/stretchr/testify/mock.(*Mock).Called(0xc0002085b0, {0xc0001bd370, 0x1, 0x1})
        C:/workspace/go/gopath/pkg/mod/github.com/stretchr/testify@v1.9.0/mock/mock.go:466 +0x125
github.com/voedger/voedger/pkg/vit/mock.(*PLogEvent).CUDs(0xc0002085a0, 0xc000069520)
        C:/workspace/go/gopath/pkg/mod/github.com/voedger/voedger@v0.0.0-20240916130329-a109ed54f540/pkg/vit/mock/impl.go:30 +0x6f
github.com/voedger/voedger/pkg/goutils/iterate.ForEachError[...](0xc000037cd8, 0xc0001bd350)
        C:/workspace/go/gopath/pkg/mod/github.com/voedger/voedger@v0.0.0-20240916130329-a109ed54f540/pkg/goutils/iterate/impl.go:35 +0x97
github.com/untillpro/airs-bp3/packages/air/vat.TestNewRestaurantVatProjector_BasicUsage.newRestaurantVatProjector.func4({0x11b98f8, 0xc0002085a0}, {0x11ba168, 0xc000208a20}, {0x11b88d0, 0xc000208a80})  
        C:/workspace/go/airs-bp3/packages/air/vat/impl_newrestaurantvatprojector.go:39 +0x2b1
github.com/untillpro/airs-bp3/packages/air/vat.TestNewRestaurantVatProjector_BasicUsage(0xc00007b860)
        C:/workspace/go/airs-bp3/packages/air/vat/impl_newrestaurantvatprojector_test.go:193 +0x2ef1
testing.tRunner(0xc00007b860, 0x116d150)
        C:/Program Files/Go/src/testing/testing.go:1690 +0xcb
created by testing.(*T).Run in goroutine 1
        C:/Program Files/Go/src/testing/testing.go:1743 +0x377
```

--- a/pkg/vit/mock/impl.go
+++ b/pkg/vit/mock/impl.go
@@ -75,6 +75,7 @@ func (b *StateKeyBuilder) PutInt32(name string, value int32)        { b.Called(n
 func (b *StateKeyBuilder) PutInt64(name string, value int64)        { b.Called(name, value) }
 func (b *StateKeyBuilder) PutString(name, value string)             { b.Called(name, value) }
 func (b *StateKeyBuilder) PutQName(name string, value appdef.QName) { b.Called(name, value) }
+func (b *StateKeyBuilder) String() string                           { return b.Called().String(0) }
 
 type StateValueBuilder struct {
 	istructs.IStateValueBuilder

